File: AngularCarRental/BestCarsRental/src/app/shared/models/order.model.ts
import { Car } from './car.model';
import { Customer } from './customer.model';

    export class Order {
        OrderID: number;
        StartDate: Date = new Date();
        ExpectedReturnDate: Date = new Date();
        ActualReturnDate: Date = new Date();

		Car: Car = new Car();
		Customer: Customer = new Customer();
    }

File: WebApi/BestCarsRental_BO/OrderModel.cs
using System;
using System.ComponentModel.DataAnnotations;


namespace BestCarsRental_BO
{
    public class OrderModel
    {
        public int OrderID { get; set; }

        [Required]
        public System.DateTime StartDate { get; set; }

        [Required]
        public System.DateTime ExpectedReturnDate { get; set; }

        [Required]
        public Nullable<System.DateTime> ActualReturnDate { get; set; }

        [Required]
        public CarModel Car { get; set; }

        [Required]
        public CustomerModel Customer { get; set; }
    }
}
